prow/README.md

```diff
@@ -1,255 +1,77 @@
-couple days, then switch `skip_report` to false.
-
-The `trigger` is a regexp that matches the `rerun_command`. Users will be told
-to input the `rerun_command` when they want to rerun the job. Actually, anything
-that matches `trigger` will suffice. This is useful if you want to make one
-command that reruns all jobs.
-
-
-### Job Environment Variables
-
-Prow will expose the following environment variables to your job. If the job
-runs on Kubernetes, the variables will be injected into every container in
-your pod, If the job is run in Jenkins, Prow will supply them as parameters to
-the build.
-
-Variable | Periodic | Postsubmit | Batch | Presubmit | Description | Example
---- |:---:|:---:|:---:|:---:| --- | ---
-`JOB_NAME` | ✓ | ✓ | ✓ | ✓ | Name of the job. | `pull-test-infra-bazel`
-`JOB_TYPE` | ✓ | ✓ | ✓ | ✓ | Type of job. | `presubmit`
-`JOB_SPEC` | ✓ | ✓ | ✓ | ✓ | JSON-encoded job specification. | see below
-`BUILD_ID` | ✓ | ✓ | ✓ | ✓ | Unique build number for each run. | `12345`
-`PROW_JOB_ID` | ✓ | ✓ | ✓ | ✓ | Unique identifier for the owning Prow Job. | `1ce07fa2-0831-11e8-b07e-0a58ac101036`
-`REPO_OWNER` | | ✓ | ✓ | ✓ | GitHub org that triggered the job. | `kubernetes`
-`REPO_NAME` | | ✓ | ✓ | ✓ | GitHub repo that triggered the job. | `test-infra`
-`PULL_BASE_REF` | | ✓ | ✓ | ✓ | Ref name of the base branch. | `master`
-`PULL_BASE_SHA` | | ✓ | ✓ | ✓ | Git SHA of the base branch. | `123abc`
-`PULL_REFS` | | ✓ | ✓ | ✓ | All refs to test. | `master:123abc,5:qwe456`
-`PULL_NUMBER` | | | | ✓ | Pull request number. | `5`
-`PULL_PULL_SHA` | | | | ✓ | Pull request head SHA. | `qwe456`
-
-Note: Examples of the JSON-encoded job specification follow for the different
-job types:
-
-Periodic Job:
-```json
-{"type":"periodic","job":"job-name","buildid":"0","prowjobid":"uuid","refs":{}}
-```
-
-Postsubmit Job:
-```json
-{"type":"postsubmit","job":"job-name","buildid":"0","prowjobid":"uuid","refs":{"org":"org-name","repo":"repo-name","base_ref":"base-ref","base_sha":"base-sha"}}
-```
-
-Presubmit Job:
-```json
-{"type":"presubmit","job":"job-name","buildid":"0","prowjobid":"uuid","refs":{"org":"org-name","repo":"repo-name","base_ref":"base-ref","base_sha":"base-sha","pulls":[{"number":1,"author":"author-name","sha":"pull-sha"}]}}
-```
-
-Batch Job:
-```json
-{"type":"batch","job":"job-name","buildid":"0","prowjobid":"uuid","refs":{"org":"org-name","repo":"repo-name","base_ref":"base-ref","base_sha":"base-sha","pulls":[{"number":1,"author":"author-name","sha":"pull-sha"},{"number":2,"author":"other-author-name","sha":"second-pull-sha"}]}}
-```
-
-## Bots home
-
-[@k8s-ci-robot](https://github.com/k8s-ci-robot) and its silent counterpart
-[@k8s-bot](https://github.com/k8s-bot) both live here as triggers to GitHub
-messages defined in [config.yaml](config.yaml). Here is a
-[command list](https://go.k8s.io/bot-commands)
-for them.
+For a brief overview of how Prow runs jobs take a look at ["Life of a Prow Job"](/prow/life_of_a_prow_job.md).
```

Comment:
nit: the link is defined at bottom already so you can drop that :-)

Reply:
The link isn't defined below, it is just used again so this needs to specify the link as well.
If we want to switch all duplicated links to reference a shared link defined at the bottom lets do that in a separate PR.